earth_enterprise/src/fusion/autoingest/AssetFactory.h

```diff
@@ -256,6 +257,199 @@ namespace AssetFactory
   }
 
 
+  template <class MutableDerivedVersionHandleType, class Version,
+            class ConfigType, class VersionDType, class Extras = int>
+  MutableDerivedVersionHandleType ReuseOrMakeAndUpdate(const std::string& ref_,
+                                                       AssetDefs::Type type_,
+                                                       const khMetaData& meta_,
+                                                       const ConfigType& config_)
+  {
+      using AssetHandleType = typename MutableDerivedVersionHandleType::Impl::MutableAssetType;
+      std::vector<SharedString> inputarg, boundInputs;
```

Comment:
I think there's a bug here because inputarg is never assigned to anything.

Reply:
After comparing with the old code this is not a bug, inputarg is supposed to be empty. We can save time by taking out the std::transform call and just declaring an empty boundInputs.